src/Symfony/Component/Messenger/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+* Add `skip` option to `messenger:failed:retry` command when run interactively to skip message and requeue it
```

Comment:
There will be no change in display but for some transport (like doctrine) the created_at will actually be updated.
I've tried to just ignore the message and go to the next one but for the DoctrineTransport the same message will be prompt immediately since message are retrieved one by one :
```php
if ($receiver instanceof ListableReceiverInterface) {
            // for listable receivers, find the messages one-by-one
            // this avoids using get(), which for some less-robust
            // transports (like Doctrine), will cause the message
            // to be temporarily "acked", even if the user aborts
            // handling the message
}